app/views/claims/email_address.html.erb

```diff
@@ -8,7 +8,7 @@
           <%= form.label :email_address, t("tslr.questions.email_address"), class: "govuk-label govuk-label--xl"  %>
         </h1>
 
-        <span class="govuk-hint">We will only use your email address to update you about your claim.</span>
+        <span class="govuk-hint">We will only use your email address to update you about your claim. We recomend you use a personal email address.</span>
```

Comment:
Typo in recommend.

Reply:
I think that was a bit of train brain, should be all good now.